build.assets/Dockerfile

```diff
@@ -71,6 +71,7 @@ RUN mkdir -p /opt && cd /opt && curl https://storage.googleapis.com/golang/$RUNT
 
 ENV GOPATH="/go" \
     GOROOT="/opt/go" \
+    GOMODCACHE="/tmp/gomodcache" \
```

Comment:
I'd put this as a `-e` flag where `/tmp` is mounted to the container, to keep like code/concerns grouped.

Reply:
Good idea. Addressed in 6d62d2d.